src/Hot.ts

```diff
@@ -2153,7 +2153,7 @@ type ParseWebSocketConnectRequestUrlResult =
 type ParseWebSocketConnectRequestUrlError =
   | {
       tag: "BadUrl";
-      expectedStart: "/?";
+      expectedStart: "/elm-watch?";
```

Comment:
I considered having something like `const WEBSOCKET_PATH = "/elm-watch"` but not sure which file to put this is so it's shared between `Hot.ts` and `client.ts`

Reply:
I would have done it like this too, it’s fine :+1: